[REV] OD-931, revert "[FIX] OD18-133, quatra_dispatching: Expose hasSelecotrUsedOffset"

StefanRijnhart opened the pull request:
This reverts commit e9c9980a6ac3f89c513fda5559a4ea44a3d15fa7.

Obsoleted by a new implementation of the row number widget.

StefanRijnhart committed:
[REV] OD-931, revert "[FIX] OD18-133, quatra_dispatching: Expose hasSelecotrUsedOffset"

This reverts commit e9c9980a6ac3f89c513fda5559a4ea44a3d15fa7.

Obsoleted by a new implementation of the row number widget.

TomAlbrechtQuatra reviewed (changes requested):
De mixin werkt goed!

Tussen de discussion en forcasted purchase weight kolom zit wel nog steeds een niet-resizable kolom. Kunnen we daar iets aan doen? :

<img width="933" height="283" alt="Image" src="https://github.com/user-attachments/assets/9d89778f-5df8-425c-9cea-a6487d393acf" />

StefanRijnhart committed:
[FIX] OD-931, web_remember_tree_column_width: fetch element from start event

Analogous to the inherited code, the event listener would work on the resize
stop event, but this event would regularly not have the same target as the
resize start event in which case the field name and size could not be
determined.
